examples/vision/yolov8.py

```diff
@@ -105,6 +113,11 @@
 object detection tasks.
 """
 
+"""shell
+!gdown 19CWdNL3ePq9XIgMeb_RjA641iGra3nWm
```

Comment:
Was something wrong with loading the data from Kaggle?

Reply:
Yes, there was issue with pycocotools...

Reply:
I'd like to avoid loading this from Drive if we can.

Can you share your Kaggle notebook where you were seeing this error? It's probably a simple fix with the dataloading, and it'd be a bit nicer than loading a Drive file where the auth + verification is not as easy.